pyproject.toml

```
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "strawberry-sqlalchemy-mapper"
 packages = [ { include = "strawberry_sqlalchemy_mapper", from = "src" } ]
-version = "0.2.0"
+version = "0.2.1"
```

```suggestion
version = "0.2.0"
```

this is updated automatically :)